### ISSUE TITLE
Not working with jest, Invalid variable access: _get__

### ISSUE THREAD

**Issue by cwagner22** [speedskater/babel-plugin-rewire#223]
`babel-plugin-rewire` is not working with jest for me, because of `babel-plugin-jest-hoist`.
```
babel-plugin-jest-hoist: The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables.
    Invalid variable access: _get__
```

### To Reproduce

Check out this [repo](https://github.com/cwagner22/react-native-unit-tests).
Run `npm run jest`

I know there is already a similar issue open but I don't understand why not more people are having this issue since the app is downloaded 70k+ times per week.
Since `babel-plugin-jest-hoist` is included in `babel-jest` I believe it should affect many people...

**Comment by bhovhannes** [comment]
Seems to be similar to #183

**Comment by cwagner22** [comment]
@bhovhannes Yes, unfortunately no solution worked for me, I had to patch `babel-plugin-jest-hoist` to whitelist `_get__`...

**Comment by VandalPaulius** [comment]
I think I have similar problem, but it say `TypeError: _get__(...) is not a function`.

**Comment by sergio-domingues** [comment]
Did anyone discover a solution for this issue?

**Comment by nGoline** [comment]
I'm having the same issue. Does anyone know how to overcome this?
@cwagner22 Can you share how you patched it?

**Comment by lancemarks** [comment]
I am having this issue. It is holding up an important PR.
